docs/en/stack/ml/rules.asciidoc

```diff
@@ -0,0 +1,39 @@
+[role="xpack"]
+[[ml-rules]]
+=== Machine learning rules and filters
+++++
+<titleabbrev>Rules and filters</titleabbrev>
+++++
+
+By default, as described in <<ml-analyzing>>, anomaly detection is unsupervised 
+and the {ml} models have no awareness of the domain of your data. As a result, 
+{ml} jobs might identify events that are statistically significant but are 
+uninteresting when you know the larger context. Machine learning rules enable 
+you to customize anomaly detection. 
+
+_Rules_ instruct anomaly detectors to change their behavior based on 
+domain-specific knowledge that you provide. When you create a rule, you can  
+specify conditions, scope, and actions. By default, the scope of a rule is 
+unlimited and you must specify at least one condition. When the conditions of a 
+rule are satisfied, its action is triggered. 
+
+For example, if you have an anomaly detector that is analyzing CPU usage, you 
+might decide you are only interested in abnormally high usage. You can define a 
```

Comment:
`abnormally high usage` takes my mind to simply using a `high` variant of a function. I think it might be better to say something like "you might decide you are only interested in anomalies where the CPU usage is greater than a certain threshold".